src/main/resources/xml/schema/2020-1/nordic2020-1.nav-references.sch

```diff
@@ -125,9 +125,9 @@
             <let name="document-in-nav" value="((preceding::html:a | self::*) intersect ancestor::html:nav//html:a)[substring-before(@href,'#') = $href][1]"/>
             <let name="document-in-nav-depth" value="count($document-in-nav/ancestor::html:li)"/>
             <let name="depth-in-nav" value="count(ancestor::html:li)"/>
-            <let name="depth-in-content" value="$result-ref-first/xs:integer((@data-heading-depth, @data-sectioning-depth)[1])"/>
+            <let name="depth-in-content" value="$result-ref-first/xs:integer(@data-heading-depth)"/>
 
-            <assert test="not($result-ref-first) or $depth-in-nav = $depth-in-content">[nordic_nav_references_2b] The nesting of headlines in the content does not match the
+            <assert test="not(@data-heading-depth) or not($result-ref-first) or $depth-in-nav = $depth-in-content">[nordic_nav_references_2b] The nesting of headlines in the content does not match the
```

Comment:
By removing `@data-sectioning-depth`, there won't be any errors for sectioning elements with `aria-label`. I think we should validate `aria-label` headings as well.

And I think it's best to split the assertion into two:

- the depth in the navigation document should be the same as the sectioning depth in the content documents
- if there is a heading element, then the heading element depth should be the same as the sectioning depth, for depths up to 6. For sectioning depths deeper than 6, the heading element depth must be 6.